odl/solvers/nonsmooth/proximal_operators.py

```diff
@@ -1494,6 +1494,67 @@ def _call(self, x, out):
     return ProximalLInfty
 
 
+def proximal_convex_conj_linfty(space):
+    """Proximal operator factory of the Linfty norm/distance convex conjugate.
+
+    Implements the proximal operator of the convex conjugate of the
+    functional ::
+
+        F(x) = ||x||_\\infty
+
+    with ``x`` in ``space``.
+
+    Parameters
+    ----------
+    space : `LinearSpace` or `ProductSpace` of `LinearSpace` spaces
+        Domain of the functional F
+
+    Returns
+    -------
+    prox_factory : function
+        Factory for the proximal operator to be initialized.
+
+    Notes
+    -----
+    The convex conjugate :math:`F^*` of the functional
+
+    .. math::
+        F(x) = ||x||_\\infty.
+
+    is in the case of scalar-valued functions given by the indicator function
+    of the unit 1-norm ball
+
+    .. math::
+        F^*(y) = \\iota_{B_1} \\big( y \\big).
```

Comment:
How about fixing the math formatting here? Would be better to have a raw string `r"""` and use single-backslash for TeX commands. Also, norm bars should be `\|`.

Reply:
Thanks for the feedback. I fixed the formatting. Any other comments?

Reply:
No I don't have any other complaints :-)

Just an observation @mehrhardt: the email that you used for the git commits doesn't seem to be associated with your Github account. You may want to do that, otherwise these commits are credited to some Matthias J. Ehrhardt that is not you :-)

Reply:
Cheers, got this sorted out now.